src/rowtransforms.jl

```diff
@@ -0,0 +1,60 @@
+struct NormalizeRow <: DataAugmentation.Transform
+	normstats
+	normcols
+end
+
+struct Categorify <: DataAugmentation.Transform
+	pooldict
+	categorycols
+end
```

Comment:
```suggestion
struct Categorify{T, S} <: DataAugmentation.Transform
	categories::T
	categorycols::S
end
```
Two changes: swap to `categories` to just be a vector of the categories. I don't think we need the complexity of categorical arrays when the mapping is just the index in a list of categories passed by the user.

Reply:
To reduce the complexity, we could just use the `catdict` used in `getcategorypool` directly for `Categorify`. A vector of vectors (or a `NamedTuple`) with the classes for each category could work as well.

Reply:
I'm not sure this will work if `categories` is just a vector of categorical column names as we'll have to replace the class for a categorical column with an integer, and for doing this we'll need information about all the classes which are present in a particular column.

Reply:
Yeah, sorry it should be a `NamedTuple`/`Dict`.